src/check-npm-package-metadata.js

```diff
@@ -27,7 +27,7 @@ const metadata = require('@adobe/aio-lib-template-validation');
     for (const failure of results.failures) {
       errors.push(`\n**${failure.description}**\n`);
       for (const error of failure.errors) {
-        errors.push(`:x: ${error.message}`);
+        errors.push(`:x: ${error.message || error}`);
```

Comment:
'@adobe/aio-lib-template-validation' produces inconsistent error messages, some of them are in a plain text, not json.